sentences/pt_br/climate_HassClimateGetTemperature.yaml

```diff
@@ -0,0 +1,8 @@
+language: pt_br
+intents:
+  HassClimateSetTemperature:
```

Comment:
I guess this file was copied from a wrong reference. This intent here should be `HassClimateGetTemperature`